client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-add-license-notification/index.tsx

```diff
@@ -39,52 +39,60 @@ export default function SiteAddLicenseNotification() {
+		return {
+			args: {
+				lastItem: lastItem,
+				selectedSite,
+				remainingItems: remainingItems.join( ', ' ),
+			},
+			components: {
+				strong: <strong />,
+				em: <em />,
+			},
+		};
+	}
+
 	return (
 		<>
 			{ assignedLicenses.length > 0 && (
 				<div className="site-add-license-notification__license-banner">
 					<Notice onDismissClick={ () => clearLicenses( 'fulfilled' ) } status="is-success">
-						{ translate(
-							'{{strong}}%(assignedLicenses)s{{/strong}} was succesfully assigned to {{em}}%(selectedSite)s{{/em}}. Please allow a few minutes for your features to activate.',
-							'{{strong}}%(assignedLicenses)s{{/strong}} were succesfully assigned to {{em}}%(selectedSite)s{{/em}}. Please allow a few minutes for your features to activate.',
-
-							{
-								count: assignedLicenses.length,
-								args: {
-									assignedLicenses: assignedLicenses
-										.map( ( license ) => license.name )
-										.join( ', ' ),
-									selectedSite,
-								},
-								components: {
-									strong: <strong />,
-									em: <em />,
-								},
-							}
-						) }
+						{
+							// We are not using the same translate method for plural form since we have different arguments.
+							assignedLicenses.length > 1
+								? translate(
+										'{{strong}}%(remainingItems)s and %(lastItem)s{{/strong}} were succesfully assigned to {{em}}%(selectedSite)s{{/em}}. Please allow a few minutes for your features to activate.',
+										getMessageArgs( assignedLicenses )
+								  )
+								: translate(
+										'{{strong}}%(lastItem)s{{/strong}} was succesfully assigned to {{em}}%(selectedSite)s{{/em}}. Please allow a few minutes for your features to activate.',
+										getMessageArgs( assignedLicenses )
+								  )
+						}
 					</Notice>
 				</div>
 			) }
 			{ rejectedLicenses.length > 0 && (
 				<div className="site-add-license-notification__license-banner">
 					<Notice onDismissClick={ () => clearLicenses( 'rejected' ) } status="is-error">
-						{ translate(
-							`An error occurred and your {{strong}}%(rejectedLicenses)s{{/strong}} wasn't assigned to {{em}}%(selectedSite)s{{/em}}.`,
-							`An error occurred and your {{strong}}%(rejectedLicenses)s{{/strong}} weren't assigned to {{em}}%(selectedSite)s{{/em}}.`,
-							{
-								count: rejectedLicenses.length,
-								args: {
-									rejectedLicenses: rejectedLicenses
-										.map( ( license ) => license.name )
-										.join( ', ' ),
-									selectedSite,
-								},
-								components: {
-									strong: <strong />,
-									em: <em />,
-								},
-							}
-						) }
+						{
+							// We are not using the same translate method for plural form since we have different arguments.
+							rejectedLicenses.length > 1
+								? translate(
+										`An error occurred and your {{strong}}%(remainingItems)s and %(lastItem)s{{/strong}} weren't assigned to {{em}}%(selectedSite)s{{/em}}.`,
```

Comment:
These strings will need really good translator comments, it isn't clear from the string alone what `remainingItems` and `lastItems` are.